.eslintrc.json

```diff
@@ -0,0 +1,35 @@
+{
+  "extends": [
+    "@dvsa/eslint-config-ts"
+  ],
+  "settings": {
+    "import/resolver": {
+      "typescript": {
+        "alwaysTryTypes": true
+      }
+    }
+  },
+  "rules": {
+    "@typescript-eslint/no-explicit-any": "off",
```

Comment:
We definitely do not :)

There's a lot of refactoring coming so allowing all the rules now will have a huge changeset. I'm trying to break the PRs down into something manageable :) More linting changes / adding of rules will be coming, but was intending to put them in subsequent PRs

Reply:
With regards to this, I know the new template we have been using utilises `Biome`, which is used for defining the linting/prettier rules. It may be a good opportunity to move towards using that in the subsequent PR's?

Reply:
Good catch! (this was written quite a ways back, so wasn't aware of biome at the time). I'll update this PR to use biome 👍

Reply:
@matthew2564 Am i right in thinking that existing repos will continue to use eslint for now?